nixos/modules/services/web-apps/code-server.nix

```
@@ -86,7 +86,7 @@ in
       hashedPassword = lib.mkOption {
         default = "";
         description = ''
-          Create the password with: `echo -n 'thisismypassword' | npx argon2-cli -e`.
+          Create the password with: {command}`echo -n 'thisismypassword' | nix run nixpkgs#libargon2 -- "$(pwgen 20 -n1)" -e`
```

```suggestion
          Create the password with: {command}`echo -n 'thisismypassword' | nix run nixpkgs#libargon2 -- '$(pwgen 20 -n1)' -e`
```
These should be single quotes, otherwise the package must be in PATH already.

---

that does not change with single quotes, you instead use the string `'$(pwgen 20 -n1)'` as the salt

---

oh, you're right. sorry

---

how about
```suggestion
          Create the password with: {command}`echo -n 'thisismypassword' | nix run nixpkgs#libargon2 -- "$(command -v pwgen >/dev/null && pwgen 20 -n1 || echo $RANDOM)" -e`
```

---

or if the salt doesn't need a cryptographically safe prng then 
```suggestion
          Create the password with: {command}`echo -n 'thisismypassword' | nix run nixpkgs#libargon2 -- "$RANDOM" -e`
```

---

How about this?
fish doesn't seem to support $RANDOM
```suggestion
          Create the password with: {command}`echo -n 'thisismypassword' | nix run nixpkgs#libargon2 -- "$(head -c 20 /dev/random | base64)" -e`
```

---

:shipit: